test/y2storage/boot_requirements_checker_x86_test.rb

```diff
@@ -137,53 +109,27 @@
       end
 
       context "with a MS-DOS partition table" do
-        let(:grub_partitions) { [] }
-        let(:boot_ptable_type) { :msdos }
-
         context "if the MBR gap is big enough to embed Grub" do
-          let(:mbr_gap_size) { 256.KiB }
-
           context "in a partitions-based proposal" do
-            let(:use_lvm) { false }
+            let(:scenario) { "dos_btrfs_with_gap" }
 
             it "does not require any particular volume" do
               expect(checker.needed_partitions).to be_empty
             end
           end
 
           context "in a LVM-based proposal" do
-            let(:use_lvm) { true }
-
             context "if the MBR gap has additional space for grubenv" do
-              let(:mbr_gap_size) { 260.KiB }
+              let(:scenario) { "dos_btrfs_lvm_enough_gap" }
 
               it "does not require any particular volume" do
                 expect(checker.needed_partitions).to be_empty
               end
             end
 
             context "if the MBR gap has no additional space" do
-              it "requires only a /boot partition" do
-                expect(checker.needed_partitions).to contain_exactly(
-                  an_object_having_attributes(mount_point: "/boot")
-                )
-              end
-            end
-          end
+              let(:scenario) { "dos_btrfs_lvm_min_gap" }
 
-          context "in an encrypted proposal" do
```

Comment:
Is it not needed a separate boot on Legacy + MSDOS + Encrypted + MBR without additional space? The proposal is requiring a separate boot now in that scenario, so: a) we should document this important change (rationale or something like that), b) the proposal should be modified to fulfill new behaviour.